en/releases/centreon-core.md

```diff
@@ -121,6 +121,13 @@ By:
 
 ## Centreon Gorgone
 
+### 21.10.1
+
+#### Bugfixes
+
+- Make Gorgone’s private key readable by centreon-gorgone only
+- Fixed restart was too long and occasionally reaches the max systemctl timeout
```

Comment:
The description used in Release Note field is better I think : 
Gorgone was too long to restart, which could cause the service to reach the systemctl timeout.